superset-frontend/src/explore/components/ControlHeader.jsx

```diff
@@ -37,13 +37,15 @@ const propTypes = {
   tooltipOnClick: PropTypes.func,
   warning: PropTypes.string,
   danger: PropTypes.string,
+  hideNativeTab: PropTypes.bool,
```

Comment:
Can we name this `skipTabFocus`?

Reply:
I was consider not affect other components except the #14574 bug one, do we need to skip  all the focus that components includes ControlHeader?

Reply:
ControlHeader is only used on the Explore page, if we want to change this focus behavior, we probably want to change it for all control headers.

Reply:
@ktmud Done, changed to skipTabFocus